gef-ui/server/utils/facility-amendments.helper.ts

```diff
@@ -23,3 +25,18 @@ export const canUserAmendIssuedFacilities = (submissionType: DealSubmissionType,
 
   return isPortalAmendmentsEnabled && userHasMakerRole && isValidSubmissionType && isValidDealStatus;
 };
+
+/**
+ * Determines if a user can amend a facility.
+ *
+ * @param facility - The facility to check.
+ * @param deal - The deal associated with the facility.
+ * @param userRoles - The roles of the user.
+ * @returns - Returns true if the user can amend the facility, otherwise false.
+ */
+export const canUserAmendFacility = (facility: Facility, deal: Deal, userRoles: Role[]) => {
```

Comment:
Should this be `userCanAmendIssuedFacility`?

The naming feels a bit off with the below function in this context, `canUserAmendIssuedFacilities` because we're only consuming one facility here?